docs/content/docs/ops/state/task_failure_recovery.md

```diff
@@ -296,7 +336,25 @@ env = StreamExecutionEnvironment.get_execution_environment(config)
 
 The cluster defined restart strategy is used. 
 This is helpful for streaming programs which enable checkpointing.
-By default, a fixed delay restart strategy is chosen if there is no other restart strategy defined.
+By default, the exponential delay restart strategy is chosen if there is no other restart strategy defined.
+
+### Default restart strategy
+
+When Checkpoint is enabled and the user does not specify a restart strategy, [`Exponential delay restart strategy`]({{< ref "docs/ops/state/task_failure_recovery" >}}#exponential-delay-restart-strategy)
+is the current default restart strategy. We strongly recommend Flink users to use the exponential delay restart strategy and set it as the default restart strategy because compared to other restart strategies, it can:
+Jobs can be retried quickly when exceptions occur occasionally, and avalanches of external components can be avoided when exceptions occur frequently. The reasons are as follows:
```

Comment:
> because compared to other restart strategies, it can: Jobs can be retried quickly ...

Maybe "because by using this strategy, jobs can be retried quickly ..."

Reply:
I updated zh doc as well.